Integrations/integration-Lastline_CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 ## [Unreleased]
-
-
-## [19.12.0] - 2019-12-10
--  
-
-## [19.9.1] - 2019-09-18
--
-
+  - Deprecate lastline v1
```

Comment:
@kirbles19 you reviewed this ?

Reply:
Deprecated. Use the Lastline v2 integration instead.